docs/proposals/PROPOSAL_I.md

```diff
@@ -0,0 +1,235 @@
+# Proposal I - compatibilities extention in platform object
+
+This proposal is born from proposal A to H.
+
+The purpose of the proposal, based on the maintainers feedback, is to address the concern maintaining standalone artifact in OCI.
+
+## image spec platform object extention
+
+### revive platform.features
+
+platform.features will be reused for image selection, refer to [proposal E](https://github.com/opencontainers/wg-image-compatibility/blob/main/docs/proposals/PROPOSAL_E.md)
+
+### add platform.compatibilities descriptor
+
+Add a new descriptor in platform object in [image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md#image-index-property-descriptions).
```

Comment:
Doesn't this still imply another artifact, it's just linked to differently, directly from a manifest list?

Reply:
The idea here is to address our concerns in current OCI specifications, avoid introduing a new standalone spec ( maintainers worried about new standard to maintain). so just add new descriptor in  manifest entry's platform objector in image index, the descriptor will point to a manifest, in which the compatibilities could reside in config(the current commit), or in a new mediatype layer(another option).
 
it can be handled quite similar to artifact, but no need to use referrers api anymore. 

the compatibilities now can be decoupled from container lifecycle，and optional for runtime engine, runtime engine can be configured to use the compatibilities or not, in sync or async order, it's easy for runtime engine to evolve.